modeling-cmds/src/def_enum.rs

```diff
@@ -575,6 +575,15 @@ define_modeling_cmd_enum! {
             pub face_id: Uuid,
         }
 
+        /// Gets the shared edge between these two faces if it exists
+        #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, ModelingCmdVariant)]
+        pub struct Solid3dGetCommonEdge {
+            /// Which object is being queried.
+            pub object_id: Uuid,
+            /// The faces being queried (must be an array of length 2)
+            pub face_ids: Vec<Uuid>,
```

Comment:
```suggestion
            pub face_ids: [Uuid; 2]
```

types > comments

Reply:
whoa.  didn't know the API could handle statically sized arrays.  :+1: